src/unix/newlib/mod.rs

```diff
@@ -160,8 +171,12 @@ s! {
     }
 
     pub struct dirent {
+        #[cfg(not(target_os = "vita"))]
         pub d_ino: ino_t,
+        #[cfg(not(target_os = "vita"))]
         pub d_type: ::c_uchar,
+        #[cfg(target_os = "vita")]
+        __offset: [u8; 88],
```

Comment:
This is a repr(Rust) struct, so this offset doesn't actually offset d_name. In fact, the compiler will preferentially place d_name at offset 0 because it has a higher natural alignment